### PR TITLE
fnott: init at 1.1.0

### LINKED ISSUE #128172: Fnott package (Wayland notification,)
Hello.
Please add this notification tool to packages repo if it's possible:

https://codeberg.org/dnkl/fnott

### PR DESCRIPTION
**Pull request by polykernel**
###### Motivation for this change
package fnott for wlroots-based compositors.
resolves https://github.com/NixOS/nixpkgs/issues/128172

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/fnott/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  pname = "fnott";
+  version = "1.1.0";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/fnott.git";
+    rev = version;
+    sha256 = "1xaz13iry3fdjhqk9xzg29kbv7hqj8dzzqxr5mhrj8f2m013jdc3";
+  };
+
+  nativeBuildInputs = [ pkg-config meson ninja scdoc wayland-protocols tllist ];
```
Comment (jonringer) [inline pkgs/applications/misc/fnott/default.nix:30]: do you mind breaking these out to 1-per-line? a little crowded

**Review by jonringer — APPROVED** [review]
otherwise lgtm (just based on diff)

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -8568,6 +8568,12 @@
     github = "polygon";
     githubId = 51489;
   };
+  polykernel = {
+    email = "81340136+polykernel@users.noreply.github.com"
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:8572]:
```suggestion
    email = "81340136+polykernel@users.noreply.github.com";
```

Comment (polykernel) [reply maintainers/maintainer-list.nix:8572]: thanks for catching this careless mistake.

File: pkgs/applications/misc/fnott/default.nix (hunk 1 above)
Comment (polykernel) [reply pkgs/applications/misc/fnott/default.nix]: I have broken the inputs into separate lines, thanks

**Comment by polykernel** [comment]
I have to be off for the night, I will work on this tomorrow.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 132968 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>fnott</li>  </ul></details>

File: pkgs/applications/misc/fnott/default.nix [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+, wlroots
+, dbus
+, fcft
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fnott";
+  version = "1.1.0";
+
+  src = fetchgit {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/fnott/default.nix:24]: this should be fetchFromGitea.

Comment (polykernel) [reply pkgs/applications/misc/fnott/default.nix:24]: thanks, I didn't know fetchFromGitea existed. I will open another PR.

**Pull request merged**